charts/amoro/values.yaml

```diff
@@ -110,6 +107,37 @@ amoroConf:
   log4j2: ~
 
 
+## Configure the container of optimizer that allow you to submit optimizer by AMS
+optimizer:
+  # flink optimizer
+  flink:
+    ## enable the flink optimizer.
+    enabled: true
+    ## container name, default is flink
+    name: ~
+    ## image information of flink optimizer.
+    image:
+      ## the image repository
+      repository: arctic163/optimizer-flink
+      ## the image tag, if not set, the default value is same with amoro image tag.
```

Comment:
Grammar: "the same as" instead of "the same with", and lost "the" here.
Actually, this was discovered by my Grammarly plugin for Chrome. Just NIT.

Reply:
fixed